CHANGELOG.md

```diff
@@ -23,6 +23,8 @@
 
 ### Maintenance and fixes
 - Fix dimension ordering for `plot_trace` with divergences ([2151](https://github.com/arviz-devs/arviz/pull/2151))
+- Fix gap for `plot_trace` with option `kind="rank_bars"` ([2180](https://github.com/arviz-devs/arviz/pull/2180))
```

Comment:
this ended up in the 0.14 section instead of the unreleased, other than that it is good to merge